dpdata/abacus/md.py

```diff
@@ -33,8 +33,17 @@ def get_coord_dump_freq(inlines):
 def get_coords_from_dump(dumplines, natoms):
     nlines = len(dumplines)
     total_natoms = sum(natoms)
-    nframes_dump = int(nlines/(total_natoms + 13))
-    
+    calc_stress = False
+    if "VIRIAL" in dumplines[6]:
+        calc_stress = True
+    else:
+        assert("POSITIONS" in dumplines[6] and "FORCE" in dumplines[6])
+    nframes_dump = -1
```

Comment:
This line is useless.

Reply:
Hi Jinzhe. I think the line is helpful in recognizing the keywords are in the right place of the file, so I kept the lines.

Reply:
If you delete `nframes_dump`, it will still be defined in the successive conditional lines.  But I think keeping this line is also okay.